x-pack/plugins/apm/common/anomaly_detection/get_preferred_service_anomaly_timeseries.ts

```diff
@@ -12,13 +12,11 @@ import { ServiceAnomalyTimeseries } from './service_anomaly_timeseries';
 
 export function getPreferredServiceAnomalyTimeseries({
```

Comment:
This function is only used by `usePreferredServiceAnomalyTimeseries`. It'll be easier to read if we just inline it.